modules/virt-about-pci-passthrough.adoc

```diff
@@ -3,8 +3,53 @@
 // * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
 
 [id="virt-about_pci-passthrough_{context}"]
-= About preparing nodes and the cluster for PCI passthrough
+= About PCI passthrough
 
-The Peripheral Component Interconnect (PCI) passthrough feature enables you to access and manage hardware devices from a virtual machine. When PCI passthrough is configured, the PCI devices function as if they were physically attached to the virtual machine.
+The Peripheral Component Interconnect (PCI) passthrough feature enables you to access and manage hardware devices from a virtual machine. When PCI passthrough is configured, the PCI devices function as if they were physically attached to the guest operating system.
 
-To configure a hardware device as a host device for PCI passthrough, create a `MachineConfig` object file and add kernel arguments to enable Input-Output Memory Management Unit (IOMMU). Bind the PCI device to the Virtual Function I/O (VFIO) driver by creating a `MachineConfig` object file. Then, add the PCI device to the HyperConverged Operator.
+Cluster administrators can expose and manage host devices that are permitted to be used in the cluster. The HyperConverged Cluster custom resource specification contains two PCI host devices which are enabled by default.
+
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  permittedHostDevices: <1>
+    pciHostDevices: <2>
+    - pciDeviceSelector: "10DE:1DB6" <3>
+      resourceName: "nvidia.com/GV100GL_Tesla_V100" <4>
+    - pciDeviceSelector: "10DE:1EB8"
+      resourceName: "nvidia.com/TU104GL_Tesla_T4" <5>
+...
+----
+<1> The host devices that are permitted to be used in the cluster.
+<2> The list of PCI devices available on the node.
+<3> The `vendor-ID` and the `device-ID` required to identify the PCI device.
+<4> The name of the first PCI host device that is enabled by default.
+<5> The name of the second PCI host device that is enabled by default.
+
+[NOTE]
+====
+You can add custom PCI host devices to be used in the cluster; however, the two default host devices cannot be removed. Disable the default devices by setting the `disabled` flag to `true`.
```

Comment:
I would add "if needed" or something like that. I mean - we think the defaults are good and useful - that why we put them there. so disable them only if needed: 
> however, the two default host devices cannot be removed. If needed, disable the default devices by setting the `disabled` flag to `true`, instead of removing them.

Reply:
Made the change